schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -53,6 +53,7 @@
               },
               {
                 "oneOf": [
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-02-01-preview/Microsoft.AppConfiguration.json#/resourceDefinitions/configurationStores" },
```

Comment:
>{ "$ref": "https://schema.management.azure.com/schemas/2019-02-01-preview/Microsoft.AppConfiguration.json#/resourceDefinitions/configurationStores" }, [](start = 18, length = 150)

we recently added a new root schema: 2019-04-01/deploymentTemplate.json. Can you add this to the new schema as well?

Reply:
Done.